PV260_Project/PresentationLayer/UI/ConfigUi.cs

```diff
@@ -0,0 +1,84 @@
+﻿using BusinessLayer.DataLoading;
+using BusinessLayer.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace PresentationLayer.UI
+{
+    internal class ConfigUi : BaseUi, IConfigUi
+    {
+        private readonly IDataDownloader _dataDownloader;
+        private readonly IFileUrlService _fileUrlService;
+
+        public ConfigUi(IDataDownloader dataDownloader, IFileUrlService fileUrlService)
+        {
+            _fileUrlService = fileUrlService;
+            _dataDownloader = dataDownloader;
+        }
+
+        public async Task Run()
+        {
+            await GenerateUi(
+                new List<MenuAction>
+                {
+                    new() { Identifier = "1", Description = "Show current file url used for automatic downloads", Action = ShowCurrentFileUrl },
+                    new() { Identifier = "2", Description = "Set file url for automatic downloads", Action = SetFileUrl },
+                    new() { Identifier = "3", Description = "List all file urls", Action = ListFileUrls },
+                });
+        }
+
+        private async Task ShowCurrentFileUrl()
+        {
+            var fileUrl = await _fileUrlService.GetLatest();
+            Console.WriteLine($"Currently set file url: {fileUrl?.Url ?? "None"}");

```

Comment:
Maybe if  `fileUrl` is `null` print something like `Url is not set yet` or smth?

Reply:
changed to `Url is not set yet`